packages/messenger/src/Messenger.ts

```diff
@@ -94,6 +94,15 @@ export type MessengerEvents<
     ? Event
     : never;
 
+/**
+ * Messenger namespace checks can be disabled by using this as the `namespace` constructor
+ * parameter, and using `string` as the Namespace type parameter.
+ *
+ * This is useful for mocking a variety of different actions/events in unit tests. Please do not
+ * use this in production code.
+ */
+export const DISABLE_NAMESPACE = 'DISABLE_NAMESPACE';
```

Comment:
Technically this name makes sense, but if I'm on another team I might feel like this is doing some magic I don't understand.

What do you think about using the name `ANY_NAMESPACE` or `ALL_NAMESPACES` instead?

Reply:
I was hesitant to use `__` because this _is_ something we want people to use. Just not in production code. I considered using the `@deprecated` directive as well, and decided against it for the same reason.

Reply:
I think the attention here might have been to choose a name that is a bit more descriptive of the thing that it does. In the sense of "this messenger can be used with any namespace / all namespaces".

Reply:
I'm tempted to go with `MOCK_ANY_NAMESPACE` because it conveys both the purpose and intended usage